index.html

```diff
@@ -35,6 +36,99 @@ <h2>
             <li><img class="M" src="Images/M.png" alt="M">
         </ul>
     </main>
-    
+    </div>
+    <div class="Works">
+        <article>
+            <h2 class="First">My Recent Works</h2>
+            
+            <img src="Images/GymFit.png" alt="GymFit">
+            <h2 id="Multi">Multi-Post Stories</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <div class="boxes white">
+                <div>css</div>
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
```

Comment:
For better practices, semantics, and to fulfill the project requirements, the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul`, and each of them a `li` instead of `div` tags 📝 Kindly fix this issue for all of the projects 🙌